src/schema/v2/user.ts

```diff
@@ -229,6 +236,40 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
       resolve: ({ receive_viewing_room_notification }) =>
         receive_viewing_room_notification,
     },
+    savedArtworksConnection: {
+      type: artworkConnection.connectionType,
+      args: pageable({}),
+      resolve: async ({ _id }, args, { savedArtworksLoader }) => {
+        if (!savedArtworksLoader) return null
+
+        const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
+        const gravityOptions = {
+          page,
+          size,
+          user_id: _id,
+          private: true,
+          total_count: true,
+          sort: "-position",
```

Comment:
This sorts by most recently added, which I think is a good default. We can add more sorting support later if needed.